packages/esm-patient-forms-app/src/clinical-form-action-button.component.tsx

```diff
@@ -17,6 +17,7 @@ const ClinicalFormActionButton: React.FC = () => {
     workspaces?.[0]?.name?.match(/clinical-forms-workspace/i) ||
     workspaces?.[0]?.name?.match(/patient-form-entry-workspace/i);
 
+  const recentlyOpenedForm = workspaces.filter((w) => w.name === 'patient-form-entry-workspace')[0];
   const isFormOpen = workspaces.filter((w) => w.name === 'patient-form-entry-workspace')?.length >= 1;
```

Comment:
```suggestion
  const workspaceIndex = workspaces.findIndex(
    (workspace) => workspace.name === clinicalFormsWorkspace || workspace.name === formEntryWorkspace,
  );
  const isActiveWorkspace = workspaceIndex === 0;
  const formEntryWorkspaces = workspaces.filter((w) => w.name === formEntryWorkspace);

  const recentlyOpenedForm = formEntryWorkspaces[0];
  const isClinicalFormOpen = formEntryWorkspaces?.length >= 1;
```

Reply:
And then to support this, we could add the following variables to `constants.ts`:

```ts
export const clinicalFormsWorkspace = 'clinical-forms-workspace';
export const formEntryWorkspace = 'patient-form-entry-workspace';
```